console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/client/util/validator/TextFieldValidator.java

```diff
@@ -70,7 +69,7 @@ public enum FieldType {
         ALPHANUMERIC("alphanumeric", "^[a-zA-Z0-9_]+$"),
         NUMERIC("numeric", "^[+0-9.]+$"),
         PACKAGE_VERSION("package_version", "^[a-zA-Z0-9.\\-\\_]*$"),
-        URL("url", "(http(s)?:\\/\\/.)?(www\\.)?[-a-zA-Z0-9@:%._\\+~#=]{2,256}\\.[a-z]{2,6}\\b([-a-zA-Z0-9@:%_\\+.~#?&//=]*)");
+        URL("url", "^(https?:\\/\\/)?(www\\.)?[-a-zA-Z0-9@:%._\\+~#=//]{1,256}\\.[a-zA-Z0-9()//]{2,6}\\b(?:[-a-zA-Z0-9()@:%_\\+.~#?&\\/=]*)$");
```

Comment:
Isn't this making use of `https ` mandatory?

Reply:
It's ok to have a URL without the `https` with this regex, in fact the first part of it have the trailing `?` character (`^(https?:\\/\\/)?`).

<img width="493" alt="Screenshot 2023-06-09 at 16 44 45" src="https://github.com/eclipse/kapua/assets/66636702/68662b09-0d88-4b94-9710-7b215eda0b01">